bin/make_run_ap_pipe_conf.py

```diff
@@ -0,0 +1,24 @@
+import os
+'''Generate configuration file for ap_pipe slurm run
+'''
+
+BASE_DIR = '/project/mrawls/hits2015/'
```

Comment:
Could we make this more generic, perhaps by taking BASE_DIR as an argv argument?